hpeOneView/resources/security/users.py

```diff
@@ -42,136 +41,78 @@ class Users(object):
         '500': {'type': 'UserAndRoles'}
```

Comment:
since type is same for all api versions, include that in __init__ function. And remove this "DEFAULT_VALUES" dictionary.
```
    def __init__(self, connection, data=None):
        super(Users, self).__init__(connection, data)
        self.__default_values = {
            'type': 'UserAndRoles'
        }
```